### PR TITLE
WRO-659: Modify enzyme to testing-library in README.md

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
Remove enzyme support and dependencies in 5.0.0.

### Resolution
* Modify README.md word `enzyme` to `testing-library`

### Links
WRO-659

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by MikyungKim** [comment]
Please update PR description

File: packages/agate/template/README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ npm remove -g eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-babel
 
 ## Installing a Dependency
 
-The generated project includes Enact (and all its libraries). It also includes React and ReactDOM.  For test writing, both Sinon and Enzyme are as development dependencies. You may install other dependencies with `npm`:
+The generated project includes Enact (and all its libraries). It also includes React and ReactDOM.  For test writing, both Sinon and @testing-library/react are as development dependencies. You may install other dependencies with `npm`:
```
Comment (MikyungKim) [inline packages/agate/template/README.md]:
```suggestion
The generated project includes Enact (and all its libraries). It also includes React and ReactDOM.  For test writing, both Jest and @testing-library/react are as development dependencies. You may install other dependencies with `npm`:

```

Comment (MikyungKim) [reply packages/agate/template/README.md]: Please replace `Sinon` with `Jest` all.

Comment (hong6316) [reply packages/agate/template/README.md]: OK

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**